pkg/resources/view_grant_acceptance_test.go

```diff
@@ -1,34 +1,31 @@
 package resources_test
 
 import (
-	"fmt"
+	"bytes"
 	"os"
 	"testing"
+	"text/template"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/stretchr/testify/require"
 )
 
 func TestAccViewGrantBasic(t *testing.T) {
-	vName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	viewName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	databaseName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
```

Comment:
Can combine viewName and databaseName to same value all in one line? just a suggestion

Reply:
viewName, databaseName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)

Reply:
that won't work though since `acctest.RandStringFromCharSet` has a single string return value

Reply:
ah I see, I thought there'd be a way to give it the same value. Forgot that's how go interprets that syntax though